tests/RumorSpreading/Test_RumorSpreading.cpp

```diff
@@ -65,12 +65,9 @@ BOOST_AUTO_TEST_CASE(RRS_Initialization) {
   // Messaging
   RRS::Message dummy_message_undefined(RRS::Message::Type::UNDEFINED, 0, 0);
   RRS::Message dummy_message_push(RRS::Message::Type::PUSH, 0, 0);
-  //Test comparison and stream oeprator
-  if (dummy_message_push == dummy_message_undefined) {
-    BOOST_TEST_MESSAGE("RRS Message comparison failed for undefined message: " << dummy_message_undefined);
-  }
-  if (dummy_message_push != dummy_message_push) {
-    BOOST_TEST_MESSAGE("RRS Message comparison failed for push message: " << dummy_message_push);
-  }
+  // Test comparison and stream oeprator
+  BOOST_CHECK_EQUAL(dummy_message_push != dummy_message_undefined, true);
```

Comment:
For me, the simplest form `BOOST_CHECK(dummy_message_push != dummy_message_undefined)` seems easier to manage.

Reply:
Done